bindings/nodejs/README.md

```diff
@@ -692,11 +692,18 @@ Defines the range of addresses to get. Defaults to `0..20` if the function isn't
 
 **Returns** the address finder instance for chained calls.
 
-#### get(): Address[]
+#### includeInternal(): AddressGetter
+
+Defines that public and internal address will be returned instead of only public addresses.
+
+**Returns** the address finder instance for chained calls.
+
+#### get(): Address[] | [Address, bool][]
 
 Performs the operation.
 
-**Returns** an array of [Address](#address) instances.
+**Returns** an array of public [Address](#address) instances or an array of arrays with an Address, bool, 
```

Comment:
Might be better to describe this by 
**Returns** an array of public [Address](#address) instances or an array of arrays with an Address and a bool,

Reply:
Will change it